Client/Frontend/Browser/URLBarView.swift

```diff
@@ -424,12 +424,10 @@ class URLBarView: UIView {
             // Clear any existing text, focus the field, then set the actual pasted text.
             // This avoids highlighting all of the text.
             locationTextField.text = ""
-            locationTextField.becomeFirstResponder()
```

Comment:
Ha - looks like testURLBarContextMenu fails in landscape but works in portrait...